docs/source/FAQ.ipynb

```diff
@@ -78,13 +78,43 @@
     "```python\n",
     "import freva\n",
     "# In order to use freva we have to tell where to get the configuration\n",
-    "freva.config(\"/path/to/the/freva/config/file.conf\")\n",
+    "freva.config(config_file=\"/path/to/the/freva/config/file.conf\")\n",
     "# Talk to your admins to get the location of the config file.\n",
     "```\n",
     "\n",
     "Please talk to your admins on where the central freva configuration is located."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "9826e767-b45c-4199-bc42-efeb747a2e82",
+   "metadata": {},
+   "source": [
+    "### How can I add my own plugins?\n",
+    "\n",
+    "If you are working with python and want to load your own plugin definitions you can also the `freva.config` to load you own plugin definitons. Assuming you have\n",
```

Comment:
"If you are working with python and want to load your own plugin definitions you can also use `freva.config` to load you own plugin definitons. Assuming you have\n",